x-pack/plugins/security_solution/public/common/mock/test_providers.tsx

```diff
@@ -61,7 +61,13 @@ export const TestProvidersComponent: React.FC<Props> = ({
   onDragEnd = jest.fn(),
   cellActions = [],
 }) => {
-  const queryClient = new QueryClient();
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        retry: false,
+      },
+    },
+  });
```

Comment:
This was causing issues with mocked API calls that throw errors. I set it here "globally" because I believe we do not need `react-query` retry functionality in Jest tests. Seems to have no impact on existing tests.

Reply:
FYI: our team does not really use this here, but we do also have similar code in our test providers here: 

https://github.com/elastic/kibana/blob/3d6dbd4ad791696fbbbca6af96617076e4b6cd32/x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx#L221-L238

I'm ok with this change, but just wanted to point out that its other teams that mostly use these.